uk.co.compucorp.civicrm.tasksassignments/js/test/services/document_test.js

```diff
@@ -81,12 +81,12 @@ define([
     }));
 
     beforeEach(function () {
-      spyOn(ContactService, 'get').and.callFake(fakePromise(contactMock.contactList));
+      spyOn(ContactService, 'get').and.callFake(fakePromise(contactMock.list()));
       spyOn(AssignmentService, 'get').and.callFake(fakePromise(assignmentMock.assignmentList));
       spyOn(ContactService, 'updateCache').and.returnValue(deferred.promise);
       spyOn(AssignmentService, 'updateCache').and.returnValue(deferred.promise);
 
-      $httpBackend.whenGET(/action=get&entity=contact/).respond({});
+      $httpBackend.whenGET(/action=get&debug=true&entity=contact/).respond({});
```

Comment:
> Currently, its set to true because in file /uk.co.compucorp.civicrm.tasksassignments/js/test/globals.js the debug: 1 is set so when running the tests it will always be true, so it wont get interfere with the local environment setup.

Got it now. Thanks.